core/templates/dev/head/domain/exploration/AnswerGroupObjectFactorySpecs.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview unit tests for answer group object factory.
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+import { AnswerGroupObjectFactory } from
+  'domain/exploration/AnswerGroupObjectFactory';
+import { OutcomeObjectFactory } from 'domain/exploration/OutcomeObjectFactory';
+
+describe('Answer Group Object Factory', () => {
+  let agof: AnswerGroupObjectFactory;
+  let oof: OutcomeObjectFactory;
+  beforeEach(() => {
```

Comment:
Add a newline above this

Reply:
@ahv15 Hey Harshit, I still don't see a new line above this.

Reply:
@Showtim3  I can see it in mine.
![Screenshot (3)](https://user-images.githubusercontent.com/52852877/72544652-c2fbc900-38ad-11ea-96d8-48851bf8589a.png)

Reply:
I had made the commit